### PR TITLE
Delete gce-1-6-1-7-upgrade-* jobs

### PR DESCRIPTION
**Pull request by fejta**
http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1 
has not passed in 233 days

@kubernetes/sig-gcp-test-failures

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -866,27 +866,6 @@
...
-      "--test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
-      "--timeout=120m",
-      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=ci/latest-1.7"
-    ],
-    "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "sig-cluster-lifecycle"
-    ]
-  },
   "ci-kubernetes-e2e-gce-1-6-1-7-upgrade-cluster-new": {
```
Comment (krzyzacy) [inline jobs/config.json]: also delete the `-cluster-new` variant as well?

Comment (fejta) [reply jobs/config.json]: Those one's aren't on the list yet

**Pull request merged**